threads/serializers.py

```
@@ -79,14 +75,25 @@ def update(self, instance, validated_data):
 
 
 
-
-
-
-
 class ReplySerializer(serializers.ModelSerializer):
+    content = serializers.CharField(required=False)
+    image = serializers.PrimaryKeyRelatedField(queryset=Image.objects.all(), required=False)
     class Meta:
```

Flake8 (7.2.0)

[error] expected 1 blank line, found 0

(E301)